src/TouristInfoCards.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const TouristInfoCards = props => {
+  return (
+    <div class="card" style={{ width: "18rem" }}>
```

Comment:
Remember that when writing React components we have to convert them to use `className` not `class`